src/oc/fileio.c

```diff
@@ -798,8 +798,8 @@ static int hoc_Load_file(int always, const char* name) {
 	/* xopen the file */
 	if (b) {
 /*printf("load_file xopen %s\n", base);*/
-		assert(strlen(base) + 50 < hoc_load_file_size_);
-		sprintf(cmd, "hoc_ac_ = execute1(\"{xopen(\\\"%s\\\")}\")\n", base);
+		assert(strlen(base) < hoc_load_file_size_);
```

Comment:
In non debug mode this won’t be executed. Isn’t there nrn_assert as in coreneuron?

Reply:
this has no side effect so should disappear with NDEBUG

Reply:
It’s true that there is no side effect but I was wondering about runtime check.

Reply:
Yes. You are right. I will put it back as nrn_assert